elixir_buildpack.config

```diff
@@ -1,4 +1,5 @@
-erlang_version=22.0
+# available versions https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/blob/master/otp-versions
+erlang_version=22.2
```

Comment:
yeah, it's kinda lame that `22.0` doesn't _exist_ ... 🤦‍♂ 
Thanks for updating. 👍